core/chain-configs/src/genesis_config.rs

```diff
@@ -164,6 +164,9 @@ impl From<&GenesisConfig> for EpochConfig {
 )]
 pub struct GenesisRecords(pub Vec<StateRecord>);
 
+/// `Genesis` has an invariant that we can't enforce due to an optimization for saving memory.
```

Comment:
Would be useful to spell *which* invariant this is.

Reply:
Done